cmd/osquery-perf/agent.go

```diff
@@ -1273,7 +1359,12 @@ func (a *agent) mdmMac() []map[string]string {
 		}
 	}
 	return []map[string]string{
-		{"enrolled": "true", "server_url": a.mdmClient.EnrollInfo.MDMURL, "installed_from_dep": "false"},
+		{
+			"enrolled":           "true",
+			"server_url":         a.macMDMClient.EnrollInfo.MDMURL,
+			"installed_from_dep": "false",
+			"payload_identifier": apple_mdm.FleetPayloadIdentifier,
```

Comment:
I think "payload_identifier" ensures it gets detected as "fleet MDM"?

Reply:
Yes, I believe that's correct. https://github.com/fleetdm/fleet/blob/fdcf255d6caf59cc8a03fc2023a023ac73d565bf/server/service/osquery_utils/queries.go#L1548-L1549